packages/trees/backend/sirius-components-collaborative-trees/src/main/java/org/eclipse/sirius/components/collaborative/trees/handlers/DefaultExpandAllTreePathHandler.java

```diff
@@ -38,6 +38,9 @@
  */
 @Service
 public class DefaultExpandAllTreePathHandler {
+
+    private static final int MAXIMUM_REACHABLE_EXPAND_DEPTH = 100;
```

Comment:
I was confused by the test `depthConsidered - startingDepth < MAXIMUM_REACHABLE_EXPAND_DEPTH`. The `MAXIMUM_REACHABLE_EXPAND_DEPTH` seem to imply that this is the *absolute* maximum depth reachable, so I expected just `depthConsidered < MAXIMUM_REACHABLE_EXPAND_DEPTH`.

From what the code actually does it seems like this is more of "how much deeper do we accept to go in one single 'Expand All' compared to the initial depth"?

Reply:
You're right it's not the maximum depth but the max depth for this expand action.
I renamed it on MAX_EXPAND_DEPTH_INCREASE